content/300-guides/050-database/25-providers/provider-planetscale

```diff
@@ -0,0 +1,44 @@
+---
+title: 'Prisma with PlanetScale'
+metaTitle: 'Prisma with PlanetScale'
+metaDescription: 'Guide on using Prisma with PlanetScale'
+tocDepth: 2
+toc: true
+---
+
+<TopBlock>
+
+Prisma and PlanetScale together provide a development arena that optimizes rapid, type-safe development of data access applications, using Prisma's ORM and PlanetScale's highly scalable MySQL-based platform.
+
+</TopBlock>
+
+## About using Prisma with PlanetScale
+
+This document discusses the concepts behind using Prisma and PlanetScale, explains some important concepts and considerations on the commonalities and differences between PLanetScale and other dtabase providers, and leads you through the process for configuring your application to implement PlanetScale.
+
+Two main benfits of using PlanetScale with Prisma are:
+
+- enterprise scalability
+    - <use Jan's content>
+    
+- support non-blocking schema changes
+    - <use Jan's content>
+    
+Watch our video about [Best Practices](https://www.youtube.com/watch?v=iaHt5_hg44c) when using Prisma with PlanetScale.
+    
+### Commonalities with other database proviers
+
+<use Jan's content>
+
+### Differences to consider
+
+<use Jan's content, cover FK since PS doesn't enforce FK, etc>
+
+### Additional considerations
+
+<add note about `db push` instead of `prisma migrate`>
+
+## How to implement Planetscale
+
+<to be written, include a ssection for using with the Platform>
```

Comment:
Note that PlanetScale can be used with Prisma ORM alone, no PDP needed (and actually less useful/needed as with other database providers as DataProxy is not needed)